Sources/SwiftSyntaxBuilder/SyntaxNodeWithBody.swift

```diff
@@ -87,3 +87,23 @@ extension EnumDecl: HasTrailingMemberDeclBlock {}
 extension ExtensionDecl: HasTrailingMemberDeclBlock {}
 extension ProtocolDecl: HasTrailingMemberDeclBlock {}
 extension StructDecl: HasTrailingMemberDeclBlock {}
+
+// MARK: - IfStmt
+// IfStmtSyntax is a special scenario as we also have the `else` body or an if-else
+// So we cannot conform to `HasTrailingCodeBlock`
+
+public extension IfStmt {
+  init(_ signature: String, @CodeBlockItemListBuilder bodyBuilder: () -> CodeBlockItemList, @CodeBlockItemListBuilder elseBuilder: () -> CodeBlockItemList? = { nil }) {
```

Comment:
What about

```suggestion
  init(_ signature: String, @CodeBlockItemListBuilder bodyBuilder: () -> CodeBlockItemList, @CodeBlockItemListBuilder `else` elseBuilder: () -> CodeBlockItemList? = { nil }) {
```

I think that would make the call-side a little nicer because the second trailing closure can be named `else` (assuming you don’t have to add backticks around it).